DevOps/Nginx/Nginx_User_Permission_Restriction.md

```
@@ -0,0 +1,161 @@
+# (NGINX)How to restrict user permission in NGinx location wise userbase access
+
+code➖
+
+server {
+listen 80;
+server_name 3.110.168.41;
+
+```
```

markdownlint-cli2 (0.17.2)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)